packages/ui/forms/verify-identity/index.tsx

```diff
@@ -0,0 +1,490 @@
+            this.setState({ generalFormError: lndrVerified.formErrors.general })
+        }
+    }
+
+    showActionSheet(title, choices, onSelect) {
+        ActionSheetIOS.showActionSheetWithOptions({
+            options: choices.map(choice => choice.name).concat(cancel),
+            title,
+            cancelButtonIndex: choices.length
+        },
+        (index) => {
+            if (index === choices.length) {
+                return
+            }
+            onSelect(choices[index])
+        })
+    }
+
+    renderPicker(choices, label: string, selection: any, changeValue: (newValue: any) => void) {
+        if (Platform.OS === 'ios') {
+            return <View>
+                <Text style={style.pickerLabel} onPress={() => this.showActionSheet(label, choices, changeValue)}>{selection && selection.name ? selection.name : label}</Text>
+                <FontAwesome style={style.blackCaretDown} name={'caret-down'} onPress={() => this.showActionSheet(label, choices, changeValue)}/>
+            </View>
+        } else {
+            return <Picker
+                selectedValue={selection} style={style.picker}
+                onValueChange={changeValue}>
+                {[<Picker.Item label={label} value='0' key={-1} />]
+                    .concat(choices.map((value, key) => 
+                        <Picker.Item label={value.name} key={key} value={value}>{selection.name}</Picker.Item>)
+                )}
+            </Picker>
+        }
+    }
+
+    render() {
+        const vertOffset = Platform.OS === 'android' ? -300 : 0
+        const { firstName, lastName, street, city, state, postCode, phone, dob, country, agreement, governmentPhoto, selfiePhoto, addressPhoto,
+            firstNameError, lastNameError, streetError, cityError, stateError, postCodeError, phoneError, dobError, generalFormError,
+            governmentPhotoType, addressPhotoType } = this.state
+
+        const govPhoto = governmentPhoto ? { uri: governmentPhoto} : require('images/upload-cloud-outline.png')
+        const selfPhoto = selfiePhoto ? { uri: selfiePhoto} : require('images/upload-cloud-outline.png')
+        const addrPhoto = addressPhoto ? { uri: addressPhoto} : require('images/upload-cloud-outline.png')
+
+        const disabled = !(firstName && lastName && street && city && state && postCode && phone && governmentPhoto && selfiePhoto && addressPhoto && agreement && governmentPhotoType.code && addressPhotoType.code)
+
+        return(
+            <View style={general.whiteFlex}>
+                <View style={general.view}>
+                    <DashboardShell text="Verify Identity" navigation={this.props.navigation} />
+                    <Button close onPress={() => this.props.navigation.goBack()} />
+                </View>
+                <KeyboardAvoidingView behavior={'padding'} keyboardVerticalOffset={vertOffset} >
+                    <ScrollView keyboardShouldPersistTaps="handled">
+                        <View style={style.form}>
```

Comment:
Need to add styles to KeyboardAvoidingView and ScrollView:
<img width="451" alt="image" src="https://user-images.githubusercontent.com/1081476/46236878-d93aea00-c335-11e8-9eae-2679f98d83ae.png">